llvm/test/tools/llvm-objdump/visualize-jumps-aarch64.s

```diff
@@ -0,0 +1,204 @@
+# RUN: rm -rf %t && split-file %s %t && cd %t
+
+// RUN: llvm-mc < input.s -triple aarch64 -filetype=obj | \
```

Comment:
Omit `< ` for `llvm-mc`

`REQUIRES: aarch64-registered-target` is needed unless you place it in ELF/AArch64